src/conversions.jl

```diff
@@ -284,6 +284,13 @@ function el32convert(x::AbstractArray{T, N}) where {T<:Union{Missing, <: Number}
     end::Array{Float32, N}
 end
 
+"""
+Takes an input `LineString` and decomposes it to points.
+"""
+function convert_arguments(PB::PointBased, linestring::LineString)
```

Comment:
I think you should just define a convert for a vector of linestrings as well.  Would that be a multilinestring or something else?

Reply:
Yeah. It would facilitate a `MultiLineString`. But it's tricky to get it to work. Since `convert_arguments` is called only once what is the right way of telling `AbstractPlotting` that one `LineString` ends here and the next starts here. The docs specified that adding `NaN` is a way, but the syntax is a bit unclear.

Reply:
You can just separate the lines using `Point2f0(NaN)`, I think